src/KafkaFlow/Middlewares/Compressor/CompressorProducerMiddleware.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace KafkaFlow.Compressor
+﻿namespace KafkaFlow
```

Comment:
The class namespaces should respect the folder hierarchy. The only exception is the configuration extension methods.

Reply:
Done